pykern/pkdebug.py

```diff
@@ -54,6 +53,11 @@
     You can match any text in the line output with a regular expression, which
     is case insensitive.
 
+    If you omit args and kwargs, `pkdp` returns its argument, of use in
+    constructions such as::
```

Comment:
Maybe instead of "of use in construction" you could say "of use when you want to inspect a value without changing the flow of the code"? Not sure my wording is any better...

You example is good.

FWIW another one I commonly use is 
```python
def foo():
...
  return pkdp(res)
```